panel/template/goldenlayout/golden_layout.html

```diff
@@ -0,0 +1,67 @@
+{% extends base %}
+
+<!-- goes in body -->
+{% block postamble %}
+<script type="text/javascript" src="//code.jquery.com/jquery-1.11.1.min.js"></script>
+<script type="text/javascript" src="//golden-layout.com/files/latest/js/goldenlayout.min.js"></script>
+<link type="text/css" rel="stylesheet" href="//golden-layout.com/files/latest/css/goldenlayout-base.css" />
+<link type="text/css" rel="stylesheet" href="//golden-layout.com/files/latest/css/goldenlayout-dark-theme.css" />
+{% endblock %}
+
+<!-- goes in body -->
+{% block contents %}
+<div class="container-fluid d-flex flex-column vh-100 overflow-hidden" id="container">
```

Comment:
Each one of these CSS classes come from bootstrap and do nothing here, same with pretty much every class you use below.